src/views/integrations/tickets-wallet-plus/pdf/pass/body/date.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * PDF Pass: Body - Event Time
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/integrations/tickets-wallet-plus/pdf/pass/body/event-time.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link https://evnt.is/1amp
+ *
+ * @since TBD
+ *
+ * @version TBD
+ */
+
+ if ( empty( $event ) ) {
```

Comment:
There's a space before the `if`.
```suggestion
if ( empty( $event ) ) {
```